<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>selecasmod</class>
 <widget class="QDialog" name="selecasmod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selección de Asiento Automático</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nuevopushButton">
       <property name="text">
        <string>&amp;Nuevo</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/Files-New-File-icon.png</normaloff>:/iconos/graf/Files-New-File-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editarpushButton">
       <property name="text">
        <string>&amp;Editar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/boligrafo_editar.png</normaloff>:/iconos/graf/boligrafo_editar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ejecutarpushButton">
       <property name="text">
        <string>E&amp;jecutar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/exec.png</normaloff>:/iconos/graf/exec.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aspiradorapushButton">
       <property name="text">
        <string>&amp;Aspirador</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/vaccum_cleaner_256.png</normaloff>:/iconos/graf/vaccum_cleaner_256.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borrarpushButton">
       <property name="text">
        <string>&amp;Borrar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportarpushButton">
       <property name="text">
        <string>E&amp;xportar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/fileexport.png</normaloff>:/iconos/graf/fileexport.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importarpushButton">
       <property name="text">
        <string>&amp;Importar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/fileimport.png</normaloff>:/iconos/graf/fileimport.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tipospushButton">
       <property name="text">
        <string>&amp;Tipos</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/system-file-manager.png</normaloff>:/iconos/graf/system-file-manager.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>selecasmod</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
